spec/services/deploy_status_checker_spec.rb

```diff
@@ -0,0 +1,133 @@
+require 'rails_helper'
+
+RSpec.describe DeployStatusChecker do
```

Comment:
I'm not sure how we have our project configured, but I like it as it allows us to use RSpec's non-monkeypatching mode where nothing is leaked into the global namespace.